frontend/include/chpl/uast/Attribute.h

```diff
@@ -161,11 +168,20 @@ class Attribute final: public AstNode {
 
     ser.write(name_);
     ser.write(numActuals_);
+    ser.write(usedParens_);
     ser.write(actualNames_);
   }
 
   DECLARE_STATIC_DESERIALIZE(Attribute);
 
+  /*
+    Returns whether the attribute was written with parens.
+    This is only needed while @unstable "msg" is going through deprecation
```

Comment:
I'd drop the part of the comment mentioning 1.30-1.31, we can just silently remove the whole method when we're ready, so no need to date ourselves.